scripts/build/release.sh

```diff
@@ -153,6 +153,16 @@ mv /tmp/version.h              src/app/contextBroker/version.h
 rm -rf CHANGES_NEXT_RELEASE
 touch CHANGES_NEXT_RELEASE
 
+# Adjust Readthedocs documentation badge. Note that the procedure is not symetric (like in version.h), as
```

Comment:
This is just a comment, but anyway ...

Readthedocs -> read-the-docs ?   (looks better IMO)
symetric -> symmetric   (or perhaps symmetrical)